### DISCUSSION TITLE
Email scanner widget configuration

### DISCUSSION THREAD

**Discussion by AlessiaC95** [YetiForceCompany/YetiForceCRM#14634] [How to] [answered]
Hi everyone,

I wanted to ask how to configure the email in order to track the emails sent / received in the appropriate widget that can be found for example in "Companies and Contacts > Contacts" or "Sales"> "Price Quotation".

![email](https://user-images.githubusercontent.com/83593181/116974879-10789100-acbf-11eb-8940-1434da89dd76.PNG)

Thank you!!

**Comment by rskrzypczak** [comment, marked as answer]
[Mail scanner configuration for ticketing process - Tutorial](https://www.youtube.com/watch?v=0JBckYE9ZuI&list=PLX0lktP3Kukg_Da-TaEBys-aON68Yy2vh&index=13)

**Comment by antomassy** [comment]
Hi,

I have the same doubt. The email are imported in contact / quotes / company oinly when i click on email in roundcube in sent email. If I send an email directly from contact page / card, email is not shown in the contact - email history widgets. I don't understand if this is correct or if I have something wrong in my configuration.

Thank you

**Reply by antcorro** [reply]
I have the same problem/doubt.
Email History widget only shows  Sent emails if I open the email in Sent fonder from "My mailbox".

Is this the behavior or do I have to configure anything more?

Thank you

**Reply by mariuszkrzaczkowski** [reply]
you do not have an e-mail scanner set up, see the documentation

**Reply by antcorro** [reply]
Hello @mariuszkrzaczkowski ,

thank you, but I have configured e-mail scanner following the documentation.
![email_scanner_conf_2_2021-12-24 Mail Scanner](https://user-images.githubusercontent.com/10244840/147373610-313fe526-02ce-45ee-b333-668b1eac796f.png)

![email_scanner_conf_2021-12-24 Mail Scanner](https://user-images.githubusercontent.com/10244840/147373611-189e7e0a-be42-4cb6-94e3-6c7cf793dbf5.png)

Mail Scanner Action, Verification and Bind CRON are Active

I just need to  view in email history the email sent to and received from Accounts and Leads.

Is more configuration required?

Thanks

**Reply by mariuszkrzaczkowski** [reply]
Yes, you need to set up folders

**Reply by antcorro** [reply]
Hello, The folders are already configured. 

![YetiForce_folders_2021-12-27 Mail Scanner](https://user-images.githubusercontent.com/10244840/147505876-a6664937-0e65-4e0a-82ec-fa278c17d08c.png)

**Comment by mariuszkrzaczkowski** [comment]
show download histories of mail messages
https://gitstable.yetiforce.com/index.php?module=OSSMailScanner&parent=Settings&view=Logs

attach 
- server configuration verification report
-  cron job list 
- full logs, including cron
